test/services/DateServiceSpec.scala

```diff
@@ -91,7 +116,7 @@ class DateServiceSpec extends TestSupport with FeatureSwitching {
   "getCurrentTaxYearStart" should {
     "return the start of the current tax year" in {
       disable(TimeMachineAddYear)
-      val expectedDate = if (TestDateService.isBeforeLastDayOfTaxYear(false)) LocalDate.of(LocalDate.now.getYear - 1, 4,6)
+      val expectedDate = if (TestDateService.isBeforeLastDayOfTaxYear(false)) LocalDate.of(LocalDate.now.getYear - 1, 4, 6)
```

Comment:
Please avoid using LocalDate.now in the test or anywhere directly, use DateService to get current date

Reply:
This was autoformatter but I can fix 👍